frontend/src/pages/HomePage.css

```
@@ -0,0 +1,366 @@
+
+.wgp-home {
+  --home-ink: #f8fafc;
+  --home-ink-muted: rgba(248, 250, 252, 0.82);
+  --home-panel: rgba(255, 253, 248, 0.96);
+  --home-panel-ink: #1f2937;
+  --home-panel-muted: #57534e;
+  --home-green: #065f46;
+  --home-green-bright: #047857;
+  --home-brass: #b45309;
+  --home-rule: rgba(6, 95, 70, 0.16);
+  min-height: 100vh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)